queries.sql

```diff
@@ -9,3 +9,32 @@ SELECT * from animals WHERE neutered IS TRUE;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg >= 10.5 AND weight_kg <= 17.3;
 
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name like '%mon%';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-1-1';
```

Comment:
- [ ] This one should be `DELETE FROM animals WHERE date_of_birth >= '2022/01/01' ;` instead of `date_of_birth > '2022-1-1';`

Reply:
@BenMukebo It's "Delete all animals born after Jan 1st, 2022.". meaning Jan 1st, 2022 is not included. If I use '>=' it includes Jan 1st, 2022.